themes/common.css

```
@@ -1019,6 +1011,52 @@
+    gap: 0.55rem;
+    flex-wrap: wrap;
+}
+
+.image-job-pending-text {
+    line-height: 1.5;
+}
+
+.image-job-spinner {
+    width: 0.9em;
+    height: 0.9em;
+    border: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)